org_preferences.go

```diff
@@ -0,0 +1,55 @@
+package gapi
+
+import (
+	"bytes"
+	"encoding/json"
+)
+
+// UpdateOrgPreferencesResponse represents the response to a request
+// updating Grafana org preferences.
+type UpdateOrgPreferencesResponse struct {
+	Message string `json:"message"`
+}
+
+// OrgPreferences fetches org preferences.
+func (c *Client) OrgPreferences() (Preferences, error) {
+	var prefs Preferences
+	err := c.request("GET", "/api/org/preferences", nil, nil, &prefs)
+	if err != nil {
+		return prefs, err
+	}
+
+	return prefs, err
```

Comment:
I'm not a big fan of leaving errors unattended, but there isn't actually any value here, we can just do:
```suggestion
	return prefs, err
```

Reply:
Same applies for other methods below.